basic-algorithms-&-datastructures/index010.js

```diff
@@ -0,0 +1,13 @@
+const newsFeed = {
+    page: "front_page"
+}
+
+/*
```

Comment:
Working with objects will be covered in depth in projects that fall before this one. Review is always good though, and I just tried to shorten the next challenges where learners build the object:

/*
An object may also contain an array or another object. For example: 

```js
const objectExample = { cats:[] }
```

After the property `page`, create an `articles` property and set it to an empty array. 
*/